nominees/covid-19-sms.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Covid-19-SMS",
+  "description": "A nodejs api to send sms msg using africatalking api for delevering information about COVID-19 virus.",
+  "website": "https://github.com/africa-covid-19-response-toolkit/awareness-sms",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/africa-covid-19-response-toolkit/awareness-sms/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3
+    }
+  ],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/africa-covid-19-response-toolkit/awareness-sms",
+  "organizations": [
+    {
```

Comment:
@rachita19082 kindly update this entry to include a website URL. Thanks.

Reply:
Resolved in commit dev@ae4e198 and dev@95a7fb0